test/unit/math/mix/mat/fun/Phi_test.cpp

```diff
@@ -0,0 +1,97 @@
+#include <stan/math/mix/mat.hpp>
+#include <gtest/gtest.h>
+#include <test/unit/math/prim/mat/vectorize/prim_scalar_unary_test.hpp>
+#include <test/unit/math/rev/mat/vectorize/rev_scalar_unary_test.hpp>
+#include <test/unit/math/fwd/mat/vectorize/fwd_scalar_unary_test.hpp>
+#include <test/unit/math/mix/mat/vectorize/mix_scalar_unary_test.hpp>
+#include <stan/math/prim/mat/fun/Phi.hpp>
+#include <test/unit/math/prim/mat/vectorize/vector_builder.hpp>
+
+/**
+ * This is the structure for testing mock function Phi (defined in the
+ * testing framework).  See README.txt for more instructions.
+ */
+struct Phi_test {
+
+  /**
+   * Redefinition of function brought in from stan::math.  The reason
+   * to do this is that it wraps it up in this static template class.
+   *
+   * This is the version that's being tested.  
+   *
+   * WARNING:  assumes that the scalar values for all instantiations
+   * (prim, rev, fwd, mix) ***have already been tested***.
+   *
+   * @tparam R Return type.
+   * @tparam T Argument type.
+   */
+  template <typename R, typename T>
+  static R apply(const T& x) {
+    using stan::math::Phi;
+    return Phi(x);
+  }
```

Comment:
I'm really confused now. Why do we need to write this apply function? How does this test framework test the vectorize framework?

It looks like this is testing the scalar version of the function. If that's the case, this test should be not be in the `mat` folder and should be in the `scal` folder.